R/read-tbl.R

```diff
@@ -0,0 +1,275 @@
+#------------------------------------------------
+#' Read MIPTools tables
+#'
+#' @description
+#' The `read_tbl_*()` family of functions is designed to read data tables
+#' generated by the software program
+#' [`MIPtools`](https://github.com/bailey-lab/MIPTools). Data is read lazily
+#' using the [`vroom` package](https://vroom.r-lib.org/index.html). Data can be
+#' filtered, retaining all rows that satisfy the conditions. To be retained, the
+#' row in question must produce a value of `TRUE` for all conditions. Note that
+#' when a condition evaluates to NA, the row will be dropped.
+#'
+#' @section Data structure:
+#' Input data must contain six rows of metadata. The metadata can vary depending
+#' on what type of file is read, but typically contains information about the
+#' location of a mutation. The remaining rows represent the data for each sample
+#' sequenced. Together, the alternate, reference, and coverage tables can
+#' provide information about mutations observed and the coverage at each site.
```

Comment:
Remove the lines about together the tables can provide info. The idea is that this family will have more functions down the line, so should not focus only on these three tables.